docs/index.md

```diff
@@ -0,0 +1,97 @@
+<p align="center">
+  <img width="100" height="100" src="https://github.com/landing-ai/landingai-python/raw/main/assets/avi-logo.png">
+</p>
+
+# Welcome to Landing.ai SDK documentation
+
+Landing.ai SDK is a set of tools to help you build computer vision applications, including
+integration with [LandingLens](https://app.landing.ai/).
+
+The SDK includes helper features to acquire, process and detect things in your images and videos, with the minimum lines of code possible.
+
+## Quick start
+
+
+### Instalation
+First, install the SDK:
+
+```bash
+pip install landingai
+```
+
+### Acquiring your first images
+After installing, you can start acquiring images from one of our image sources.
+
+For example, from a single image file:
+
+```py
+from landingai.pipeline.frameset import FrameSet
+
+frame = FrameSet.from_image("/path/to/your/image.jpg") # (1)!
+frame.resize(width=512, height=512) # (2)!
+frame.save_image("/tmp/resized-image.png") # (3)!
+```
+
+1. We support a large variety of image files formats. Just give it a try.
+2. Resize the image to 512x512 pixels.
+3. Save the resized image to `/tmp/resized-image.png`.
+
+
+You can also extract frames from your webcam, for example:
+
+```py
+from landingai.pipeline.image_source import Webcam
+
+with Webcam(fps=0.5) as webcam:  # (1)!
+    for frame in webcam:
+        frame.resize(width=512, height=512) # (2)!
+        frame.save_image("/tmp/webcam-image.png") # (3)!
+```
+
+1. Capture images from the webcam at 0.5 frames per second (1 frame every 2 seconds), closing the camera at the end of the `with` block
+2. Resize the frame to 512x512 pixels
+3. Save the images as `/tmp/webcam-image-<timestamp>.png`
```

Comment:
this <timestamp> is a suffix we always add?
if so we need to add this in the other snippets doc

Reply:
Nope. The previously merged PR fixes this, but I forgot to fix in the docs. Changing it now.